akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala

```diff
@@ -96,13 +85,18 @@ object DistributedPubSubSettings {
  * @param removedTimeToLive Removed entries are pruned after this duration
  * @param maxDeltaElements Maximum number of elements to transfer in one message when synchronizing
  *   the registries. Next chunk will be transferred in next round of gossip.
+ * @param mediatorBufferSize Maximum number of messages buffered by mediator to topic actors that are in process of being terminated.
+ * @param perTopicActorBufferSize Maximum number of messages buffered by each [[akka.cluster.pubsub.DistributedPubSubMediator.Internal.Topic]]
+ *   actor to group actors that are in process of being terminated.
  */
 final class DistributedPubSubSettings(
   val role: Option[String],
   val routingLogic: RoutingLogic,
   val gossipInterval: FiniteDuration,
   val removedTimeToLive: FiniteDuration,
-  val maxDeltaElements: Int) extends NoSerializationVerificationNeeded {
+  val maxDeltaElements: Int,
+  val mediatorBufferSize: Int,
+  val perTopicActorBufferSize: Int) extends NoSerializationVerificationNeeded {
```

Comment:
this is not binary backwards compatible, you need to add a constructor that matches the old constructor